pytorch_lightning/loops/utilities.py

```diff
@@ -62,6 +64,20 @@ def _extract_hiddens(training_step_output: STEP_OUTPUT, truncated_bptt_steps: in
     return hiddens
 
 
+# rank_zero_warning of type UserWarning for max_epochs if not set
+def _parse_max_epochs_and_steps(
+    max_steps: int, max_epochs: int, max_time: Union[str, timedelta, Dict[str, int]], min_steps: int, min_epochs: int
```

Comment:
I would make the order with min* first, followed by max*, i.e.: 

min_steps, max_steps, min_epochs, max_epochs, max_time

Reply:
Gotcha!